src/rpc/methods/sync.rs

```diff
@@ -71,6 +75,26 @@ impl RpcMethod<0> for SyncState {
     }
 }
 
+pub enum SyncStatus {}
+impl RpcMethod<0> for SyncStatus {
+    const NAME: &'static str = "Filecoin.SyncStatus";
```

Comment:
```suggestion
    const NAME: &'static str = "Forest.SyncStatus";
```

Maybe better to avoid `Filecoin.` prefix since this is not a Filecoin API, what do you think? @LesnyRumcajs @lemmih

Reply:
@hanabi1224 I missed it actually, it was mentioned [here](https://github.com/ChainSafe/forest/issues/4929#issuecomment-2684489348) we need to use `Forest.SynsStatus`

Reply:
I have renamed the RPC to `Forest.SyncSnapshotProgress`.